sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/FormRecognizerServiceVersion.java

```diff
@@ -11,7 +11,8 @@
 public enum FormRecognizerServiceVersion implements ServiceVersion {
     V2_0("v2.0"),
     V2_1_PREVIEW_1("v2.1-preview.1"),
-    V2_1_PREVIEW_2("v2.1-preview.2");
+    V2_1_PREVIEW_2("v2.1-preview.2"),
```

Comment:
in .NET we only maintain the latest preview version as we encourage people to move.
Is this something Java usually does?

Reply:
yes, in Java we maintain an enum of all the supported versions and have it defaulted to the latest currently.